### ISSUE TITLE
[Bug] Restore SMS/MMS on Samsung app

### ISSUE THREAD

**Issue by cyberavis** [NeoApplications/Neo-Backup#661]
**Description**
During to restore SMS/MMS when SMS app is Samsing default masseges – version 11.5.50.13 I get an error „c: OandBackupX not default SMS/MMS app.”

**Steps To Reproduce**

1. Make backup of $ SMS/MMS JSON
2. Restore the backup

**Expected behavior**
To be able to restore.

**Log**
`{"logDate":"2022-10-11T16:23:43.494","deviceName":"crownlte","sdkCodename":"10","cpuArch":"arm64-v8a","logText":"c: OAndBackupX not default SMS/MMS app."}`

**System Information(please complete the following information):**

- Device: Samsung Galaxy S7 (herolte)
- Android Version: 10
- ROM: FloydQ V7.0 (with all hotfix)
- App's Version: 8.1.3

**Comment by tsiflimagas** [comment]
As the error indicates, you have to make NB the default SMS app to be able to restore your messages.

**Comment by cyberavis** [comment]
Yes, you have right. So I suggest to change this error to info: "Set Neo Backup as default SMS/MMS application to restore backup. After restore, please change it back to SMS/MMS application, otherwise you will not receive messages.". Then everyone will know what's going on.
